### PR TITLE
docs: add Electron 30 blog post

### PR DESCRIPTION
**Pull request by clavin**
This PR adds a new blog post for Electron 30. @electron/wg-releases, @electron/wg-outreach

Merge target: April 16th, after 30.0.0 releases.

⚠️ Do not merge until the following are completed ⚠️

* [x]   update node, v8 and chromium versions from final chrome roll under Stack Changes section
* [x]   edit link for M124 "New In Chrome" blog post
* [x]   add a few bullets for New Features section
* [x]   add any missing items in Breaking Changes section
* [x]   update End of Support

_Note: The "Check Blog links" job is going to fail until the Chrome 122 announcement blog comes out, a fail there is expected until about Tuesday_

### CODE REVIEW

File: blog/electron-30-0.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+## Notable Changes
+
+### Highlights
+
+* ASAR Integrity fuse now supported on Windows
+  * Existing apps with ASAR Integrity enabled may not work on Windows if not configured correctly.
+  * Take a look at our [ASAR Integrity tutorial](https://www.electronjs.org/docs/latest/tutorial/asar-integrity) for more information.
+* Added `WebContentsView` and `BaseWindow`, deprecating & replacing `BrowserView`
+* Added `net` module to utility process
+* Implemented support for the File System API
```
Comment (VerteDinde) [inline blog/electron-30-0.md:31]: I'd add the PR numbers/links here so people can click through and view them

```suggestion
* ASAR Integrity fuse now supported on Windows [#40504](https://github.com/electron/electron/pull/40504)
  * Existing apps with ASAR Integrity enabled may not work on Windows if not configured correctly.
  * Take a look at our [ASAR Integrity tutorial](https://www.electronjs.org/docs/latest/tutorial/asar-integrity) for more information.
* Added `WebContentsView` and `BaseWindow`, deprecating & replacing `BrowserView` [#35658](https://github.com/electron/electron/pull/35658)
* Added `net` module to utility process [#40017](https://github.com/electron/electron/pull/40017)
* Implemented support for the File System API [#41827](https://github.com/electron/electron/commit/cf1087badd437906f280373decb923733a8523e6)
```

**Review by VerteDinde — APPROVED** [review]
Looks great, thanks so much for putting it together! 🙇‍♀️

File: blog/electron-30-0.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+See [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#allow) for
+more information.
+
+#### Removed: The `--disable-color-correct-rendering` switch
+
+This switch was never formally documented but it's removal is being noted here regardless. Chromium itself now has better support for color spaces so this flag should not be needed.
+
+#### Behavior Changed: `BrowserView.setAutoResize` behavior on macOS
+
+In Electron 30, BrowserView is now a wrapper around the new [WebContentsView](api/web-contents-view.md) API.
```
Comment (VerteDinde) [inline blog/electron-30-0.md:60]: I think lint is failing here - we can add the full link:
```suggestion
In Electron 30, BrowserView is now a wrapper around the new [WebContentsView](https://www.electronjs.org/docs/latest/api/web-contents) API.
```

Comment (dsanders11) [reply blog/electron-30-0.md]: This is still failing due to the current SNAFU with deploying the latest docs to the website - it's a legit 404.

File: blog/electron-30-0.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+## Notable Changes
+
+### Highlights
+
+* ASAR Integrity fuse now supported on Windows [#40504](https://github.com/electron/electron/pull/40504)
+  * Existing apps with ASAR Integrity enabled may not work on Windows if not configured correctly.
+  * Take a look at our [ASAR Integrity tutorial](https://www.electronjs.org/docs/latest/tutorial/asar-integrity) for more information.
+* Added `WebContentsView` and `BaseWindow`, deprecating & replacing `BrowserView` [#35658](https://github.com/electron/electron/pull/35658)
+* Added `net` module to utility process [#40017](https://github.com/electron/electron/pull/40017)
+* Implemented support for the File System API [#41827](https://github.com/electron/electron/commit/cf1087badd437906f280373decb923733a8523e6)
```
Comment (erickzhao) [inline blog/electron-30-0.md]:
```suggestion
* Implemented support for the [File System API](https://developer.mozilla.org/en-US/docs/Web/API/File_System_API) [#41827](https://github.com/electron/electron/commit/cf1087badd437906f280373decb923733a8523e6)
```

File: blog/electron-30-0.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+Cross-origin iframes must now specify features available to a given `iframe` via the `allow`
+attribute in order to access them.
+
+See [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#allow) for
+more information.
+
+#### Removed: The `--disable-color-correct-rendering` switch
+
+This switch was never formally documented but it's removal is being noted here regardless. Chromium itself now has better support for color spaces so this flag should not be needed.
```
Comment (erickzhao) [inline blog/electron-30-0.md:56]: nit: grammar

```suggestion
This switch was never formally documented but its removal is being noted here regardless. Chromium itself now has better support for color spaces so this flag should not be needed.
```

File: blog/electron-30-0.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+#### Behavior Changed: cross-origin iframes now use Permission Policy to access features
+
+Cross-origin iframes must now specify features available to a given `iframe` via the `allow`
+attribute in order to access them.
+
+See [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#allow) for
+more information.
+
+#### Removed: The `--disable-color-correct-rendering` switch
```
Comment (erickzhao) [inline blog/electron-30-0.md:54]: nit: wording

```suggestion
#### Removed: The `--disable-color-correct-rendering` command line switch
```

File: blog/electron-30-0.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+If you have any feedback, please share it with us on [Twitter](https://twitter.com/electronjs) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Highlights
+
+* ASAR Integrity fuse now supported on Windows [#40504](https://github.com/electron/electron/pull/40504)
+  * Existing apps with ASAR Integrity enabled may not work on Windows if not configured correctly.
+  * Take a look at our [ASAR Integrity tutorial](https://www.electronjs.org/docs/latest/tutorial/asar-integrity) for more information.
+* Added `WebContentsView` and `BaseWindow`, deprecating & replacing `BrowserView` [#35658](https://github.com/electron/electron/pull/35658)
```
Comment (erickzhao) [inline blog/electron-30-0.md]:
```suggestion
* Added [`WebContentsView`](https://www.electronjs.org/docs/latest/api/web-contents-view) and [`BaseWindow`](https://www.electronjs.org/docs/latest/api/base-window) main process modules, deprecating & replacing `BrowserView` [#35658](https://github.com/electron/electron/pull/35658)
```

File: blog/electron-30-0.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+* Added `net` module to utility process [#40017](https://github.com/electron/electron/pull/40017)
```
Comment (erickzhao) [inline blog/electron-30-0.md]:
```suggestion
* Added [`net`](https://www.electronjs.org/docs/latest/api/net) module to utility process [#40017](https://github.com/electron/electron/pull/40017)
```

File: blog/electron-30-0.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+If you have any feedback, please share it with us on [Twitter](https://twitter.com/electronjs) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Highlights
+
+- ASAR Integrity fuse now supported on Windows [#40504](https://github.com/electron/electron/pull/40504)
+  - Existing apps with ASAR Integrity enabled may not work on Windows if not configured correctly.
+  - Take a look at our [ASAR Integrity tutorial](https://www.electronjs.org/docs/latest/tutorial/asar-integrity) for more information.
+- Added `WebContentsView` and `BaseWindow`, deprecating & replacing `BrowserView` [#35658](https://github.com/electron/electron/pull/35658)
```
Comment (erickzhao) [inline blog/electron-30-0.md:29]: We should point to https://www.electronjs.org/docs/latest/tutorial/web-embeds to compare and contrast `WebContentsView` as well.

Comment (erickzhao) [reply blog/electron-30-0.md:29]: issue(blocking): I don't think the release notes are clear enough about the BrowserView implementation being a JavaScript shim for WebContentsView now.

File: blog/electron-30-0.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+The Electron team is excited to announce the release of Electron 30.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Twitter](https://twitter.com/electronjs) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Highlights
+
+- ASAR Integrity fuse now supported on Windows [#40504](https://github.com/electron/electron/pull/40504)
+  - Existing apps with ASAR Integrity enabled may not work on Windows if not configured correctly.
```
Comment (erickzhao) [inline blog/electron-30-0.md:27]: We should give people a one-liner upgrade path without clicking through

```suggestion
  - Existing apps with ASAR Integrity enabled may not work on Windows if not configured correctly. Apps using Electron packaging tools should upgrade to `@electron/packager@18.3.1` or `@electron/forge@7.4.0`.
```

**Review by erickzhao — CHANGES_REQUESTED** [review]

**Comment by panther7** [comment]
[Changelog](https://github.com/electron/electron/releases/tag/v30.0.0) say, that BaseView is implemented in v29, but isn't truth.

File: blog/electron-30-0.md (hunk 8 above)
Comment (erickzhao) [reply blog/electron-30-0.md]: I guess "deprecating and replacing" sounds a bit vague to me as in the old impl doesn't exist in the code anymore.

File: blog/electron-30-0.md [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+- Added a new method `BrowserWindow.isOccluded()` to allow apps to check occlusion status. ([#38982](https://github.com/electron/electron/pull/38982))
+- Added proxy configuring support for requests made with net module from utility process. ([#41417](https://github.com/electron/electron/pull/41417)) <span style="font-size:small;">(Also in [28](https://github.com/electron/electron/pull/41744), [29](https://github.com/electron/electron/pull/41416))</span>
+- Added support for Bluetooth ports being requested by service class ID in `navigator.serial`. ([#41734](https://github.com/electron/electron/pull/41734)) <span style="font-size:small;">(Also in [29](https://github.com/electron/electron/pull/41735))</span>
+- Added support for `NODE_EXTRA_CA_CERTS`. ([#41822](https://github.com/electron/electron/pull/41822))
```
Comment (erickzhao) [inline blog/electron-30-0.md:52]:
```suggestion
- Added support for the Node.js [`NODE_EXTRA_CA_CERTS`](https://nodejs.org/api/cli.html#node_extra_ca_certsfile) CLI flag. ([#41822](https://github.com/electron/electron/pull/41822))
```

File: blog/electron-30-0.md [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+- V8 `12.4`
+
+Electron 30 upgrades Chromium from `122.0.6261.39` to `124.0.6367.49`, Node from `20.9.0` to `20.11.1`, and V8 from `12.2` to `12.4`.
+
+### New Features
+
+- Added a `transparent` webpreference to webviews. ([#40301](https://github.com/electron/electron/pull/40301))
+- Added a new instance property `navigationHistory` on webContents API with `navigationHistory.getEntryAtIndex` method, enabling applications to retrieve the URL and title of any navigation entry within the browsing history. ([#41662](https://github.com/electron/electron/pull/41662)) <span style="font-size:small;">(Also in [29](https://github.com/electron/electron/pull/41661))</span>
+- Added a new method `BrowserWindow.isOccluded()` to allow apps to check occlusion status. ([#38982](https://github.com/electron/electron/pull/38982))
+- Added proxy configuring support for requests made with net module from utility process. ([#41417](https://github.com/electron/electron/pull/41417)) <span style="font-size:small;">(Also in [28](https://github.com/electron/electron/pull/41744), [29](https://github.com/electron/electron/pull/41416))</span>
```
Comment (erickzhao) [inline blog/electron-30-0.md:50]:
```suggestion
- Added proxy configuring support for requests made with the `net` module from the utility process. ([#41417](https://github.com/electron/electron/pull/41417)) <span style="font-size:small;">(Also in [28](https://github.com/electron/electron/pull/41744), [29](https://github.com/electron/electron/pull/41416))</span>
```

**Review by erickzhao — COMMENTED** [review]
I think in the past we haven't had the backport indicators ("Also in XX") in the release blog. 🤔

File: blog/electron-30-0.md [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+- Node `20.11.1`
+  - [Node 20.11.1 notes](https://nodejs.org/en/blog/release/v20.11.1/)
+- V8 `12.4`
+
+Electron 30 upgrades Chromium from `122.0.6261.39` to `124.0.6367.49`, Node from `20.9.0` to `20.11.1`, and V8 from `12.2` to `12.4`.
+
+### New Features
+
+- Added a `transparent` webpreference to webviews. ([#40301](https://github.com/electron/electron/pull/40301))
+- Added a new instance property `navigationHistory` on webContents API with `navigationHistory.getEntryAtIndex` method, enabling applications to retrieve the URL and title of any navigation entry within the browsing history. ([#41662](https://github.com/electron/electron/pull/41662)) <span style="font-size:small;">(Also in [29](https://github.com/electron/electron/pull/41661))</span>
```
[the remark this answers is not in the document]
Comment (clavin) [reply blog/electron-30-0.md:48]: It looks like this release note is pretty accurate to how the API is meant to be used. It's a little bit unclear given the new `navigationHistory` module that exposes a class that can't be instantiated by devs, but the description in the release note is the intended use of this new feature by devs.

**Review by erickzhao — APPROVED** [review]

**Pull request merged**